libobs/obs-cocoa.m

```diff
@@ -138,6 +138,14 @@ static void log_processor_cores(void)
 	     os_get_physical_cores(), os_get_logical_cores());
 }
 
+static void log_emulation_status(void)
+{
+#ifndef __aarch64__
```

Comment:
What’s the reason for this ifdef? I think this should always be logged, no matter the build arch.

Reply:
I don't think it makes sense to log Rosetta or anything related to emulation when emulation is guaranteed not gonna happen. Both Rosetta and Windows X64 emulation works one way and one way only, and both companies are currently having no plan to make it work both ways. If we are ARM64 binaries already, then there is really no need to log it.

If we want to know are we ARM64 binaries or not, then we should do it somewhere else that is not here.

Reply:
In my view it very much does make sense to always have that log line - logs about system information should just always be there so that issues (especially on Mac where you can't mix translated and native) can be detected at first glance, and you don't have to check somewhere else on why the log line isn't where you expected (or isn't there at all).

Reply:
I suggest that we can do it in `libobs/obs-cocoa.m`, either in `log_os()` or `log_kernel_version()`. I think we should detect the host machine in this way, just like what I did with Windows in `libobs/obs-windows.c`.

Reply:
Going back to the original review here, I also agree with @gxalpha that emulation status should always be logged regardless of compiled architecture.